src/login-components/LoginForm.tsx

```diff
@@ -1,32 +1,33 @@
-import { useState } from 'react';
+import { useState } from "react";
 import UserInput from "../components/common/UserInput";
 import SubmitButton from "../components/common/SubmitButton";
 import { Link, useNavigate } from "react-router-dom";
 import { inputLabelClass, inputFieldClass } from "../components/common/styles";
 
-
 export default function LoginForm() {
-  const[loginError, setLoginError] = useState<boolean>(false);
+  const [loginError, setLoginError] = useState<boolean>(false);
   const navigate = useNavigate();
 
   const handleLogin = <T extends React.FormEvent<HTMLFormElement>>(e: T) => {
```

Comment:
Why are we using the generic T type here? Seems this is always just returning a form no?